dotcom-rendering/src/components/ProductCarouselCard.tsx

```diff
@@ -0,0 +1,145 @@
+import { css } from '@emotion/react';
+import {
+	headlineBold20,
+	headlineMedium17,
+	lifestyle,
+	space,
+	textSans17,
+	textSansBold15,
+	textSansBold17,
+} from '@guardian/source/foundations';
+import type { ArticleFormat } from '../lib/articleFormat';
+import type { ProductBlockElement } from '../types/content';
+import { ProductCardButtons } from './ProductCardButtons';
+import { ProductCardImage } from './ProductCardImage';
+
+export type ProductCarouselCardProps = {
+	product: ProductBlockElement;
+	format: ArticleFormat;
+	showReadMore?: boolean;
+};
+
+const baseCard = css`
+	display: flex;
+	width: 280px;
+	flex-direction: column;
+	align-items: flex-start;
+	padding: ${space[3]}px 10px ${space[4]}px 10px;
```

Comment:
```suggestion
	display: flex;
	flex-direction: column;
	align-items: flex-start;
	padding: ${space[3]}px 0 ${space[4]}px 0;
```
As these properties change depending on the breakpoint they are determined by the carousel rather than the card itself